src/Telemetry/AzureDevOpsTimeline/BuildLogScraper.cs

```diff
@@ -15,10 +15,13 @@ public class BuildLogScraper : IBuildLogScraper
     private readonly ILogger<BuildLogScraper> _logger;
     private readonly IClientFactory<IAzureDevOpsClient> _azureDevOpsClientFactory;
 
-    // Example: Environment: windows-latest
-    private static readonly Regex _azurePipelinesRegex = new Regex(@"Environment: (\S+)");
-    // Example: Image: build.ubuntu.1804.amd64
-    private static readonly Regex _oneESRegex = new Regex(@"Image: (\S+)");
+    // Example: Image: windows-latest
+    private static readonly Regex _azurePipelinesRegex = new Regex(@"Image: (\S+)");
+    // Example: 2023-04-04T15:46:13.8907100Z SKU: Standard_D4a_v4
+    //          2023-04-04T15:46:13.8907217Z Image: windows.vs2019.amd64
+    // or       2023-04-04T15:10:06.5649938Z SKU: Standard_D4a_v4
+    //          2023-04-04T15:10:06.5650033Z Image: 1es-windows-2022
+    private static readonly Regex _oneESRegex = new Regex(@"SKU:.+\n.+Image: (\S+)");
```

Comment:
Just for your awareness, there is a multiline regexp option. This wouldn't change the functionality here, but it might be good to include to declare the intent. https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expression-options#multiline-mode

Reply:
I'll give it a read, but it shouldn't be needed anymore